tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,9 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
+        self.manifest_url_app = None
```

Comment:
I think we don't need it it was for this one https://github.com/mermi/gaia/commit/cff4044e428541f283129d91cc89893825bb49f0#diff-06f57e3ac9df83884ad662cda7599c3bR166